docs/docs/gatsby-cli.md

```diff
@@ -27,11 +27,60 @@ You can also use the `package.json` script variant of these commands, typically
 
 ### `new`
 
+#### Usage
```

Comment:
Does this need updating in Contentful too?

Reply:
I don't _think_ these docs live in Contentful 🤔 Either way we should ship this for now and (worst-case scenario) update Contentful afterwards 👍 

cc @meganesu